playkit/src/main/java/com/kaltura/playkit/Utils.java

```diff
@@ -203,4 +213,48 @@ public static boolean equals(Object a, Object b) {
     public static String format(String format, Object... args) {
         return String.format(Locale.ROOT, format, args);
     }
+
+    public static String getNetworkClass(Context context) {
```

Comment:
Some of the constants are only available in latest SDKs, won't it cause a crash?

Reply:
tested this code with 4.1.2 device do you think it is enough?

Reply:
tested and seems fine you cannot print it but it is fine in run time